### PR TITLE
Updates to the CFP & schedule pages.

### PR DESCRIPTION
**Pull request by judy2k**
Still working on the content of the CFP page.

### CODE REVIEW

File: _pages/speaking.md [hunk 1]
```diff
@@ -1,104 +1,59 @@
...
+Keep an eye on our [Twitter](https://twitter.com/djangoconeurope) account for notifications!    
 
 {% comment %}
-Our <a href="{{ site.cfp_application }}">Call for Proposals (CFP)</a> is now open!
-Submit your talk or tutorial proposal by the end of day June 10th, 2022 [AoE](https://time.is/compare/0000_10_June_2022_in_Anywhere_on_Earth), and encourage your friends and colleagues to do the same.
-
-Need help with your proposal? We've got mentors and helpful tips on our [Speaker Resources](/speaking/speaker-resources/) page!
-
-<a href="{{site.cfp_application}}" class="button">Submit a Proposal</a>
+**The deadline is Feb 3rd [AoE](https://time.is/compare/0000_3_March_2023_in_Anywhere_on_Earth) 🔥** That is Feb 4th, 2023, 12:00 UTC. You can edit your submissions up until this time.
```
Comment (carltongibson) [inline _pages/speaking.md:13]: The text says Feb 3rd but IIUC the link is for March 3rd

File: _pages/speaking.md [hunk 2]
```diff
@@ -1,104 +1,59 @@
...
-Please see our <a href="/covid/">COVID-19 Policy</a> for more information.
-
-{% comment %}
-At this time, we are planning on most presenters being in San Diego to present their talks in person. Things may always change depending on how the pandemic progresses over the year.
-
-If you are unable to travel to the conference, please submit your talk with a note that you will be only be able to present a pre-recorded talk online. We will have a handful (less than 10) of online talks that we'll air during breaks in the in-person schedule and more.
+[Submit your talk here!](https://pretalx.evolutio.pt/djangocon-europe-2022/cfp){:target="_blank"}
 {% endcomment %}
 
 ## Proposing to DjangoCon US
```
Comment (hypha) [inline _pages/speaking.md:18]: /s/US/Europe

File: _pages/speaking.md [hunk 3]
```diff
@@ -1,104 +1,59 @@
...
+* Pushing Django to its limits
+* Fundamentals, in a modern light.
+* The Django and Python community, culture, history, past, present & future, the why the who and the what of it all
+* Security
+* Whatever you deem appropriate – it's your conference, after all!
 
-* [DjangoCon US 2021 Talks](https://2021.djangocon.us/talks/)
-* [DjangoCon US 2019 Talks](https://2019.djangocon.us/talks/)
-* [DjangoCon US 2018 Talks](https://2018.djangocon.us/talks/)
+You may think that your idea is not suited to DjangoCon Europe, or that you couldn't possibly present it well, or that others will do it better. **We want to prove you wrong**. You should consider reading [Sasha's blog post](https://web.archive.org/web/20190625135013/https://www.mxsasha.eu/blog/2015/03/11/why-you-should-speak/){:target="_blank"} on why you should speak at DjangoCon Europe! This blog post still holds true 6 years later.
```
Comment (hypha) [inline _pages/speaking.md:38]: s/6/8

File: _pages/speaking.md [hunk 4]
```diff
@@ -1,104 +1,59 @@
...
 
-### Deep Dive Talks (May 31stth)
+If you would like to ask a question, you are always welcome to write the programme committee: [content@djangocon.eu](mailto:content@djangocon.eu)
```
Comment (hypha) [inline _pages/speaking.md:40]: write "to"

File: _pages/speaking.md (hunk 2 above)
Comment (judy2k) [reply _pages/speaking.md:18]: I can't see where it says US!

**Comment by judy2k** [comment]
@carltongibson @hypha 😛 

image

File: _pages/speaking.md (hunk 1 above)
Comment (rebkwok) [reply _pages/speaking.md:13]: In the news posts, I've made it end of 26th Feb anywhere on earth (as we discussed, 26th Feb makes the closing date a Sunday, and then we can assume we might extend for a week to 5th March).
https://github.com/djangocon/2023.djangocon.eu/pull/18/files#diff-8640e7df84f1f4b2c31ade71b9dcd70516f787cfc8768e01c42abb2fd41b5d27R12
I quite like the idea of allowing submissions as long as it's still 26th Feb somewhere, but we can fix a different time - just as long as we make it the same in all the places :slightly_smiling_face:

**Review by rebkwok — APPROVED** [review]

**Pull request merged**